### DISCUSSION TITLE
Clarify that "Neurodesk play" allows to try neurodesk out

### DISCUSSION THREAD

**Discussion by civier** [neurodesk/neurodesk.github.io#109] [Ideas]
I'm not sure that everybody will understand that "Neurodesk play" in https://neurodesk.github.io/ allows to try the system out.
I would put this button first in line, and change the name to something like "Try out Neurodesk in the browser". Maybe it doesn't fit in a button, but you got the idea. Alternatively, "Interactive Online Demo" might do the job as well.

**Comment by stebo85** [comment]
agree - will think about a better way.

**Reply by aswinnarayanan** [reply]
I think the "Get Started" quick links is getting overcomplicated with the new services.
We could create a section underneath for the Google Colab, Nectar and Neurodesk Play.... as they may need a bit of description.

I'll draft something up

**Reply by aswinnarayanan** [reply]
Does this look okay https://aswinnarayanan.github.io/neurodesk.github.io/

**Reply by stebo85** [reply]
yes, that's perfect :)

**Comment by civier** [comment]
I kind of feel that people would like to know a bit more about Neurodesk before installing it, so I would put a link to the FAQ just below the platform buttons. Something like "Read more about Neurodesk", and next to it I would put the "Neurodesk play" button, clarifying it's an interactive demo.
Also, because the FAQ is so good, I would put it in the top menubar. I know that it's in the left side menu, but because we don't have a "contact us" button on the top bar, the FAQ would be a good substitute (I also added a link to the discussion form from the FAQ section in a separate pull request, so if somebody doesn't find answers in the FAQ, the discussion forum will be the next stop)
Other than that, the re-design looks very good.

**Reply by aswinnarayanan** [reply]
Done

**Comment by civier** [comment]
Thanks. Looks great!
